internal/testutil/bus.go

```diff
@@ -0,0 +1,93 @@
+package testutil
+
+import (
+	"github.com/golang/mock/gomock"
+	"github.com/pokt-network/pocket/p2p/providers/current_height_provider"
+	"github.com/pokt-network/pocket/p2p/providers/peerstore_provider"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/shared/messaging"
+	"github.com/regen-network/gocuke"
```

Comment:
```suggestion
	"github.com/pokt-network/gocuke"
```

Now you have published releases should these update?

Reply:
Good question! Short answer, no.

We would prefer to use the upstream modules but need to be able to use our fork until those changes are merged upstream. To facilitate this we can use the replace directive (above) so that the import statements in the code don't need to be updated when we switch.